.github/workflows/ci.yaml

```diff
@@ -59,11 +64,6 @@ jobs:
       - name: Generate pipeline definitions
         run: poetry run pre-commit run gen-docs-components --all-files --show-diff-on-failure
 
-      # TODO: enable when PEP 604 incompatibilty is in typer is resolved https://github.com/tiangolo/typer/issues/348
```

Comment:
is this enabled now in Ruff? I believe Typer still doesn't support the new type union syntax

Reply:
It is not, but I have excluded only that particular rule from pyupgrade

Reply:
https://github.com/bakdata/kpops/pull/363/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R96